app/controllers/CacheController.scala

```diff
@@ -56,6 +57,24 @@ class CacheController @Inject() (
         }
   }
 
+  def fetchSubmittedLinkedLrn(lrn: String): Action[AnyContent] = authenticate().async {
+    implicit request =>
+      cacheRepository
+        .get(lrn, request.eoriNumber)
```

Comment:
Could potentially look to expore making a new respository endpoint that only returns the LRN and isSubmitted field. May reduce weight compared to retrieving the whole UA. (though this is minimal, only refactor if you have time)

Reply:
will do it in a new branch for all connector calls Matt.

Reply:
This only returning those 2 fields Matt. however, while querying mongo.. you just want to fetch those 2 fields instead of all UserAnswers?